packages/@webex/plugin-meetings/src/member/index.ts

```diff
@@ -294,6 +307,8 @@ export default class Member {
       this.isAudioMuted,
       this.type
     );
+
+    this.mediaStatus = MemberUtil.extractMediaStatus(this.participant);
```

Comment:
Question: Looking at the code I can see that processMember() is only called from the constructor, but it seems that we create new members with every update? Just wanted to double check: has this been tested and proven to handle Locus DTO updates correctly?